configs/gn_baselines/e2e_mask_rcnn_R_50_FPN_1x_gn.yaml

```diff
@@ -1,5 +1,5 @@
 INPUT:
-  MIN_SIZE_TRAIN: 800
+  MIN_SIZE_TRAIN: (800,)
```

Comment:
Thanks for fixing those!
Quick question: the permissions of those files have been changed. Was that on purpose? Is one preferred over the other?

Reply:
The permissions are changed incidentally. Will revert them back.